Bluebird is no longer in use in Knex

Since `knex@0.20.11`, Bluebird is no longer in use

So it produce the following error: `Cannot find module 'bluebird' from 'node_modules/knex-hooks/src/run-hooks.js'` with:
* knex@0.21.1
* knex-hooks@0.0.8

--- a/package.json
+++ b/package.json
@@ -26,6 +26,7 @@
   },
   "license": "ISC",
   "dependencies": {
+    "bluebird": "^3.7.2",
     "co": "^4.6.0",
     "lodash.isfunction": "^3.0.8",
     "lodash.remove": "^4.7.0",

